### PR TITLE
[fmt] Guesser

### PR DESCRIPTION
**Pull request by RMeli**
----
📚 Documentation preview 📚: https://mdanalysis--4851.org.readthedocs.build/en/4851/

### CODE REVIEW

File: testsuite/MDAnalysisTests/guesser/test_base.py [hunk 1]
```diff
@@ -33,76 +33,126 @@
...
+            np.array([np.nan, np.nan, np.nan, np.nan], dtype=np.float64)
+        )
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                names,
+                masses,
+            ],
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/guesser/test_base.py]:
```suggestion
            ]
```
Trailing comma messing things up.

File: testsuite/MDAnalysisTests/guesser/test_base.py [hunk 2]
```diff
@@ -33,76 +33,126 @@
...
+        names = Atomnames(np.array(["C", "H", "H", "O"], dtype=object))
+        types = Atomtypes(np.array(["1", "2", "3", "4"], dtype=object))
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                names,
+                types,
+            ],
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/guesser/test_base.py]: Trailing comma.

File: testsuite/MDAnalysisTests/guesser/test_base.py [hunk 3]
```diff
@@ -33,76 +33,126 @@
...
-        assert_allclose(u.atoms.masses, np.array(
-            [0, 1.00800, 1.00800, 0]), atol=0)
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                types,
+                masses,
+            ],
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/guesser/test_base.py]: Trailing comma.

File: testsuite/MDAnalysisTests/guesser/test_base.py [hunk 4]
```diff
@@ -33,76 +33,126 @@
...
-        assert_allclose(u.atoms.masses, np.array(
-            [12.01100, 1.00800, 1.00800, 15.99900]), atol=0)
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                types,
+                masses,
+            ],
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/guesser/test_base.py]: Trailing comma.

File: testsuite/MDAnalysisTests/guesser/test_base.py [hunk 5]
```diff
@@ -33,76 +33,126 @@
...
+        names = Atomnames(np.array(["C", "H", "H", "O"], dtype=object))
+        types = Atomtypes(np.array(["", "", "", ""], dtype=object))
+        top = Topology(
+            4,
+            1,
+            1,
+            attrs=[
+                names,
+                types,
+            ],
```
Comment (RMeli) [inline testsuite/MDAnalysisTests/guesser/test_base.py]: Trailing comma.

**Comment by RMeli** [comment]
I fixed a few nasty trailing commas, but otherwise LGTM.

**Review by orbeckst — APPROVED** [review]
blanket LGTM — please merge when you're happy with it @RMeli 

🎄 🎁

**Pull request merged**